### DISCUSSION TITLE
v2.10.0

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#1324] [Versions]
This version adds AppleSilicon support and adds support for [Bühler](https://www.buhlergroup.com/global/de/products/roastmaster_coffeeroaster.html) and [Joper](https://joper-roasters.com/) roasting machines.

---
Release Sponsor: [Paolo Scimone Coffee Consulting](https://www.paoloscimone.com/)
<a target="_blank" href="https://www.paoloscimone.com/">
<img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/paoloscimone-logo.jpg?raw=true" width="500"></a>
---

The [Artisan project](https://artisan-scope.org/) runs on donations from individuals and companies recognising the value of Artisan for their work or leisure. Consider supporting this project with your [donation](https://www.paypal.com/paypalme/MarkoLuther), or even better, purchase an [artisan.plus membership](https://shop.artisan.plus/)!

## Documentation

__>>>  [Release Notes](https://artisan-roasterscope.blogspot.com/2023/11/artisan-v2100.html) <<<__

See the [detailed release history](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md) for a complete list of additions, changes and fixes and the [Installation Instructions](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) for platform specific notes. Read our [Quick-Start Guide](https://artisan-scope.org/docs/quick-start-guide/)!

## Supported Platforms

Builds are available for
* macOS 12 Monterey and higher incl. Sonoma macOS 14 (Intel & AppleSilicon)
* Windows 10 and 11 (x64)
* Ubuntu 22.04+ (GCC11)
* Red Hat 8.4+ (GCC10)
* experimental: Raspberry Pi OS 12 bookworm 64bit

Legacy builds for macOS High Sierra 10.13 and Windows 8 (x64) are available too.

---

PLEASE FOLLOW AND TAG US!  
&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.facebook.com/ArtisanScope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/facebook-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.instagram.com/artisanscope/"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/instagram-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/youtube-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://masto.ai/@artisan"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/mastodon-icon-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.linkedin.com/company/artisan-plus"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/linkedin-svgrepo-com.svg?raw=true" width="30"></a>
---

__Licence Notes__

_Artisan is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version._

_Artisan is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE._

_Copies of the GNU General Public License has been included with this 
distribution in the file `LICENSE.txt`. An online version is available at
<http://www.gnu.org/licenses/>._

<hr /><em>This discussion was created from the release <a href='https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.10.0'>v2.10.0</a>.</em>

**Comment by secuspec** [comment]
Artisan Scope Version 2.8.4 runs perfectly on Windows 11 with the Aillio Bullet. So there seems something broken with the latest version. Only visible difference during the installation is, that with version 2.10.0 Microsoft Visual C++ 2015-22022 Redistributable (x64) - 14.38.33130 appears at the end of the installer.

I receive „No backend“ when pressing ON or Start.

**Comment by wingnutsglory** [comment]
Upon installation of Win version of v2.10.0, antivirus software identified the following:

![image](https://github.com/artisan-roaster-scope/artisan/assets/153400482/01d9daf4-aae3-4558-9d76-9ca1a6545ecc)

**Reply by roasterdave** [reply]
This is a false positive. 

Artisan itself is open source and Artisan in turn uses additional open source libraries. Nothing is proprietary. That means all of the source code is available for public inspection. 

Sometimes the same libraries used by Artisan are used by bad folks when they create malware. That may cause some antivirus program(s) to pattern match on some malicious software that uses the same library to then flag Artisan. 

You did not identify the antivirus product that gave this message. **Please report to them that this is a false positive.** Most reputable antivirus vendors provide an online form to make such a report after which they will make an investigation and update their program. 

Note that Microsoft Defender passes Artisan v2.10.0 release.

There is nothing more the Artisan team can do.

**Reply by wingnutsglory** [reply]
Yeah, this was with Bitdefender.  Thanks for the response and I'll see if I can submit a report.

**Reply by wingnutsglory** [reply]
I have submitted a report.

**Reply by roasterdave** [reply]
Thanks for making that report.  
Please try the [continuous build](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous).  As of this writing, [VirusTotal ](https://www.virustotal.com/gui/file/4d9c582b11edb3fb19432dde8846912dcb6b96d44cec9ed6f84021703334272d) does not show BitDefender flagging artisan.exe.  Would you confirm that it starts properly for you?

**Comment by MAKOMO** [comment]
Loring support is broken on v2.10. Fixed in current [continuous builds](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous).

**Comment by mo3adz** [comment]
I own a Joper roaster and it contains PLC ( micrologix1400) I'm trying to connect it to Artisan software using the right IP address ( I got it from the PLC it self) and the software doesn't respond. It gave me an error message as following failed to connect ) .

**Reply by MAKOMO** [reply]
Is the IP address of your Joper reachable over the network from your PC running Artisan? Could you successful ping the machine from the PC? A "failed to connect" simply means that the IP packages send by Artisan did not reach its destination. Thus either you did not configure your PC for a direct local connection on the same IP network or not both, the machine and your PC are well connected via a router.
